store/helper/helper.go

```diff
@@ -309,7 +309,11 @@ func NewFrameItemFromRegionKey(key []byte) (frame *FrameItem, err error) {
 	frame.TableID, frame.IndexID, frame.IsRecord, err = tablecodec.DecodeKeyHead(key)
 	if err == nil {
 		if frame.IsRecord {
-			_, frame.RecordID, err = tablecodec.DecodeRecordKey(key)
+			var handle kv.Handle
+			_, handle, err = tablecodec.DecodeRecordKey(key)
```

Comment:
How to use `DecodeRecordKey`  for this, the user expects that the `key` to be decoded into `cluster format`, but receives an `IntHandle`, and the error returned is nil or vice versa

Reply:
This PR only do the refactor, the handle is always IntHandle.